app/controllers/forms_controller.rb

```diff
@@ -1,8 +1,15 @@
 class FormsController < ApplicationController
   after_action :verify_authorized, except: :index
   after_action :verify_policy_scoped, only: :index
+
   def index
-    @forms = policy_scope(Form) || []
+    if @current_user.super_admin?
+      @search_form = Forms::SearchForm.new({ organisation_id: @current_user.organisation_id }.merge(search_params))
```

Comment:
I'm not sure I understand why we've created a form class here. Why is this the best way to get the organisation ID out?